CHANGELOG.md

```diff
@@ -1,3 +1,6 @@
+## 2024-04-24, Version 20.5.0 (Stable), @mislam987
+* Add hint property to checkboxes to fix spacing issue
```

Comment:
rather than to 'fix spacing issue' I would say something  along the lines of 'in line with govuk design guidelines' - the hint property wasn't there before so it isn't a fix it is something we are adding in to align with govuk design guidelines (it just happened to be picked up as a spacing issue in nrm when updating nrm)

Reply:
I've updated the wording in the changelog now.